packages/sui-cz/config.js

```diff
@@ -3,13 +3,13 @@ const readdirSync = require('fs').readdirSync
 const statSync = require('fs').statSync
 
 const basePath = process.cwd()
-const package = require(path.join(basePath, 'package.json'))
+const packageConfig = require(path.join(basePath, 'package.json')).config
```

Comment:
But in the is accessing the config inside the config? Is that right? https://github.com/SUI-Components/sui/pull/3/files/2e1548d79021427651f226ae1189b2082f473b6b#diff-47efdbae34d9cf15c07739301961b08eR10